CMakeLists.txt

```diff
@@ -595,13 +600,15 @@ endif()
 # The avif_internal target should not be used by external code.
 if(BUILD_SHARED_LIBS)
     add_library(avif_internal STATIC ${AVIF_SRCS})
-    # Copy most properties from the public avif library target.
+    # Copy or duplicate most properties from the public avif library target.
```

Comment:
"Copy" and "duplicate" have similar meanings, so it's not clear what this comment is trying to differentiate by using these two words.

By looking at the diffs, I infer that this comment considers lines 604-605 as copying (because they use `TARGET_PROPERTY:avif`) and considers lines 607-608 as duplicating (because they don't use `TARGET_PROPERTY:avif` but rather "duplicate" lines 585-586 with the library name changed). Is that correct?

Line 606 is not a duplicate of line 584. Why is that?

Reply:
> `TARGET_PROPERTY:avif`) and considers lines 607-608 as duplicating (because they don't use `TARGET_PROPERTY:avif` but rather "duplicate" lines 585-586 with the library name changed). Is that correct?

Yes

> Line 606 is not a duplicate of line 584. Why is that?

Because internal target does not need to differentiate build and install interfaces.